src/rgw/rgw_iam_policy.cc

```diff
@@ -1492,7 +1501,7 @@ ostream& operator <<(ostream& m, const Statement& s) {
 
   if (!s.conditions.empty()) {
     m << "Condition: ";
-    print_array(m, s.conditions.cbegin(), s.conditions.cend());
+    print_dict(m, s.conditions.cbegin(), s.conditions.cend());
```

Comment:
and see http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition.html